custom_model_runner/datarobot_drum/drum/language_predictors/python_predictor/python_predictor.py

```diff
@@ -58,6 +58,17 @@ def predict(self, input_filename):
 
         return predictions
 
+    def transform(self, input_filename):
+        start_predict = time.time()
+        transformed_df = self._model_adapter.transform(input_filename, model=self._model)
+        end_predict = time.time()
+        execution_time_ms = (end_predict - start_predict) * 1000
+
+        # TODO: call monitor only if we are in structured mode
```

Comment:
needs ticket

Reply:
(@yakov-g or someone? this todo was already there when I got here....)

Reply:
Please remove TODO; we have separate methods for predict and predict_unstructured

Reply:
So, I think monitoring is not needed here.
MLOps monitoring is used to report predictions metrics, so we should report predictions only from predict().
Please remove self.monitor() call from here.
Also please remove TODO from the predict() call

Reply:
sounds good

Reply:
I agree.